openapi/how-to-generate-the-bundled-openapi-definition.md

```diff
@@ -0,0 +1,41 @@
+# How to generate the bundled OpenAPI definition
+
+1. Install `java` and the `swagger-codegen-cli`
```

Comment:
what about providing a script that do all of this work?

Reply:
Yes, that's what I'm trying to do as the next step.